```
2024-07-14 14:37:58 ----------------Check results--------------------
2024-07-14 14:37:58 paddle.incubate.autotune.set_config:1
2024-07-14 14:37:58 <string>:15:37: error: Argument 1 to "set_config" has incompatible type "Dict[str, object]"; expected "Union[_ConfigKernel, str, None]"  [arg-type]
2024-07-14 14:37:58 Found 1 error in 1 file (checked 1 source file)
2024-07-14 14:37:58 
2024-07-14 14:37:58 
2024-07-14 14:37:58 >>> Mistakes found in type checking!
2024-07-14 14:37:58 >>> Please recheck the type annotations. Run `tools/type_checking.py` to check the typing issues:
2024-07-14 14:37:58 > python tools/type_checking.py paddle.incubate.autotune.set_config
```

看样子得 ignore 下

--- a/python/paddle/incubate/autotune.py
+++ b/python/paddle/incubate/autotune.py
@@ -95,7 +95,7 @@ def set_config(config: _ConfigKernel | str | None = None) -> None:
             ...         "enable": True,
             ...     }
             >>> }
-            >>> paddle.incubate.autotune.set_config(config)
+            >>> paddle.incubate.autotune.set_config(config) # type: ignore[arg-type]
 
             >>> # config is the path of json file.
             >>> config_json = json.dumps(config)

